README.md

```diff
@@ -193,124 +242,119 @@ Other acceptable location examples:
 
-### System tags (eyeball networks and more)
+You can also define your own "world" location, for example:
 
-Many probes are going to be tagged by our system. At the moment this includes:
+- `from germany, greece, uk, usa, canada, japan, china, south africa --limit 20`
 
-- Cloud regions for Google Cloud and AWS. e.g. `aws-eu-west-1` and `gcp-us-south1` They follow each provider's naming scheme and are prefixed by their name.
-- Eyeball and datacenter networks. e.g. `eyeball-network` and `datacenter-network` Unlike datacenter hosted probes, eyeball network are probes hosted with ISP providers that offer internet access to regular people and small businesses. 
+returns probes from these countries in roughly equal proportions to meet the limit of 20.
 
-### No UUIDs
+>[!Important]
+>The limit parameter is global and applies to the location as a whole, not allowing the API to return more than the limit. If you want to set custom limits per location, use our API directly instead of the "magic" field.
 
-Our probes don't expose any unique IDs to target. The recommended way to narrow down your results is to use filters and even stack them together.
+### Leverage system tags (eyeball networks and more) 🏷️
+Many probes are going to be tagged by our system. At the moment, this includes:
 
-This way popular locations such as `from Amsterdam` or `from AWS` will be automatically load-balanced among multiple probes in the same location and avoid overloading any specific one.
+- **Google Cloud and AWS cloud region names**. For example, `aws-eu-west-1` and `gcp-us-south1`. These tags follow the respective provider's naming scheme and are prefixed with their name.
+- **Eyeball and data center network tags**. `eyeball` and `datacenter`. `eyeball` probes are hosted with ISPs that provide internet access to regular people and small businesses. As the name suggests, `datacenter` tags are intended for probes hosted in a data center.
 
-### Best effort results
+### Things to keep in mind
 
-When requesting a specific number of probes there is no guarantee that the API will respond back with the exact same amount. Keep this in mind and use it to your benefit.
+#### Probes share no UUIDs
+Our probes don't expose unique IDs that you can use to target them explicitly. Our probes don't expose unique IDs that you can use to target them explicitly. Instead, we recommend fine-tuning the probe selection by using and combining filters or specifying IDs from previous measurements, as described in the best practice section above.
 
-### Probe availability
+This ensures that popular locations, like `from Amsterdam` or `from AWS`, are automatically load-balanced across multiple probes within the same location, preventing the overload of specific probes.
 
-Globalping only exposes and allows users to interact with online probes. A probe is considered online and becomes available via our API when:
+#### Test result variability
```

Comment:
I dont think anyone will understand what the title means and will have to read the description. The previous title made it clear immediately what this is about